NerdDinner/Models/Dinner.cs

```diff
@@ -63,8 +58,8 @@ public IEnumerable<RuleViolation> GetRuleViolations()
 
     public class RuleViolation
     {
-        public string ErrorMessage { get; private set; }
-        public string PropertyName { get; private set; }
+        public string ErrorMessage { get; }
```

Comment:
In both these cases, the private setter is unnecessary. From outside the class, the field is considered read-only either way. From inside the class, the only difference is that a private setter can be accessed anywhere in the class, and a read-only field can only be modified in the constructor. Since these are only modified in the constructor anyway, no need for a private setter.